### ISSUE TITLE
GameInput stops working on Host machine when connected to with RDP until rebooted

### ISSUE THREAD

**Issue by TriumphRick** [microsoft/GDK#28]
GameInput appears to stop working on the target machine when any user remote desktops to it, even after the remote session is disconnected, until the affected machine is rebooted. This can be reproduced as follows:

- Run the Gamepad sample from the Xbox-GDK-Samples repo in Gaming.Desktop.x64
- Verify GameInput is working properly
- Have a user on any other computer remote desktop to the machine you just verified GameInput to be working
- Disconnect the remote session
- Run the Gamepad sample from the Xbox-GDK-Samples repo again
- Verify GameInput is no longer receiving any input: It will detect controllers, but it will seemingly never pass any input to applications, neither through GetCurrentReading nor GetNextReading

**Comment by FLY1NGSQU1RR3L** [comment]
@TriumphRick  - just read through your report and will be pinging a few folks to see if this is a new issue.  I might tie this over on the issues for the sample itself as the folks who work on GameInput aren't paying as close attention to feedback on the GDK installer point.

**Comment by FLY1NGSQU1RR3L** [comment]
@TriumphRick  - are you in our developer program? If so, can you raise this on the Xbox developer forums (https://forums.xboxlive.com) so we can pair you with an expert?

**Comment by TriumphRick** [comment]
Yes we are. Figured this was the more appropriate forum considering it was publically released, but I've cross-posted this issue on the XBox Developer Forum now for visibility. Thanks. :)

**Comment by Squall-Leonhart** [comment]
This sounds similar to something i'm experiencing too, except theres no RDP involved, WGI based apps stop receiving input on Account X when you sign into and then log out of Account Y on the switch user screen.

Signing out and back into account X resolves it.

Applications like Xbox Accessories, and apps using recent versions of SDL2 input are affected too as they use WGI. (The former can still vibrate the controller tho)

It would be reasonable to assume the simple creation of a second user session on the machine causes the faulty behavior.

@TriumphRick rather than restarting, does simply logging your user off and back on restore the control behavior?

**Comment by walbourn** [comment]
Can you confirm if this is fixed or not in https://www.nuget.org/packages/Microsoft.GameInput/2.0.26100.5334?

**Comment by lsalvadormsft** [comment]
Closing this bug due to age and as the issue should be resolved with the latest version of the API. Please let us know if you are still reproducing the issue in your end with the latest v3.1 version of the API.

**Comment by Squall-Leonhart** [comment]
This is NOT fixed.
